ofrak_core/ofrak/model/tag_model.py

```diff
@@ -1,21 +1,25 @@
 import functools
-from collections import defaultdict
-from typing import Set, Iterable, Tuple
+from typing import Iterable, Set, Tuple, Optional, cast, List
 
 
 class ResourceTag(type):
-    @functools.lru_cache(None)
```

Comment:
Did you intentionally delete `@functools.lru_cache` from `tag_specificity`?

Reply:
Yes, because the caching is within that function now (`cls._specificity` check)